server/src/main/webapp/META-INF/context.xml

```diff
@@ -1,2 +1,6 @@
 <!-- Candlepin needs symlink support to read its jars -->
-<Context path="/candlepin" allowLinking="true" />
+<Context path="/candlepin" allowLinking="true">
+    <Loader className="org.apache.catalina.loader.VirtualWebappLoader"
+            virtualClasspath="${basedir}/target/classes;${basedir}/target/${project.build.finalName}/WEB-INF/lib;/usr/lib64/jss/jss4.jar"/>
+    <JarScanner scanAllDirectories="true"/>
+</Context>
```

Comment:
Please add a newline at the end of the file here.

Reply:
Also, I have a small concern about `/usr/lib64/`.  In some systems (non-64 bit systems) the path is often `/usr/lib`.  If we are only supporting x86_64, no problem, but we need to check if that's true.  If we're supporting other arches, we'll need to figure something out.

Reply:
Added newline. Changed path to /usr/lib/java/jss4.jar which seems consistent on RHEL 6-8 for both 64 and 32-bit versions.